### PR TITLE
chore: Use NVIDIA IPAM Plugin by default

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

**Review by ykulazhenkov — APPROVED** [review]

File: example/crs/nvidia.com_v1alpha1_ nv_ipam_ippool_cr.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# 2024 NVIDIA CORPORATION & AFFILIATES
```
Comment (rollandf) [inline example/crs/nvidia.com_v1alpha1_ nv_ipam_ippool_cr.yaml:1]: Do you have a space in the file name here?

**Review by rollandf — APPROVED** [review]

File: example/crs/nvidia.com_v1alpha1_ nv_ipam_ippool_cr.yaml (hunk 1 above)
Comment (e0ne) [reply example/crs/nvidia.com_v1alpha1_ nv_ipam_ippool_cr.yaml:1]: yes :(, it's fixed now

**Review by rollandf — APPROVED** [review]

**Review by ykulazhenkov — APPROVED** [review]

**Review by adrianchiris — APPROVED** [review]

File: example/rdma-gpu-test-pod1.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ metadata:
     k8s.v1.cni.cncf.io/networks: rdma-net-ipam
```
Comment (krembu) [inline example/rdma-gpu-test-pod1.yml:6]: @e0ne seems like a tyope and change should be on this line

**Comment by krembu** [comment]
@e0ne  if we default nv-ipam for vanilla kubernetes i think we should enable it by default in openshift

**Pull request merged**